url6da19b5a7a_krakrak_Adventure_Game_tgz-pJ8-adventure_MainJ8/src/commands/Examine.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package commands;
+
+import NPC.Npc;
+import items.*;
+import java.util.List;
+
+/**
+ * Describes the Item in parameters
+ * @author Irakiza Krak
+ */
+public class Examine extends AbstractCommand{
+    //private String param;
+    boolean contains = false;
+    boolean containsNpc = false;
+    //Default Constructor
+    /**
+     * Sets the name and the description of the command
+     */
+    public Examine(){
+        super("EXAMINE", "Describes the object typed after the command, if it exists","exa");
+    }
+    /**
+     * Verifies that the object is in the room or in backpack before giving description
+     * @param game
+     * @param params 
+     */
+    @Override
+    public void execute(Game game, String params) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 22. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.